lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -106,27 +106,3 @@ def test_404_microsites(self):
         # check response with branding
         resp = self.client.get(url, HTTP_HOST=settings.MICROSITE_TEST_HOSTNAME)
         self.assertContains(resp, settings.MICROSITE_CONFIGURATION['test_site']['email_from_address'])
```

Comment:
I see some MICROSITE_CONFIGURATION code up here - that's OK to stay?

Reply:
Good catch.  I need to remove that test as well.  _(I thought I had removed both.)_